notebooks-auth/resources/user_test_cowbird_jupyter.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Check the user's <code>notebooks</code> directory, which should be a symlink to the user's Jupyterhub notebook directory.
</pre><p>Each comment like this (for this cell and following ones) could be a markdown cell on their own, since they mark distinct checks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>